ampersand-form-view.js

```diff
@@ -91,12 +93,11 @@ module.exports = View.extend({
         }
     },
 
-    checkValid: function (forceFire) {
-        var valid = this._fieldViewsArray.every(function (field) {
+    checkValid: function () {
+        this.valid = this._fieldViewsArray.every(function (field) {
```

Comment:
You can instantly return assignment here `return this.valid = this.fieldViewsArray.every...`.